src/index.html

```diff
@@ -15,30 +17,28 @@ <h1>LEADERBOARD</h1>
 
```

Comment:
## PR Title
![pr-title](https://user-images.githubusercontent.com/93039359/180228984-6f85d55c-8196-4a02-8aae-8689859823a8.png)
**[Optional]**
- [ ] Your PR has a title that is not so descriptive enough. Because **Get api** doesn't tell you why you are getting the api. PR title fully describes what is happening in this pull request. You could use **Leaderboard: hit the API** for your PR title. :+1: